data_structures/src/chain.rs

```diff
@@ -262,6 +263,31 @@ impl Default for Signature {
     }
 }
 
+impl Signature {
+    /// Serialize the Signature for interoperability with OpenSSL.
+    pub fn to_bytes(&self) -> Result<[u8; 65], failure::Error> {
+        match self {
+            Secp256k1(x) => {
+                let signature = Secp256k1_Signature::from_der(x.der.as_slice())
+                    .map_err(|_| Secp256k1ConversionError::FailSignatureConversion)?;
+
+                let v = match x.der[1] {
```

Comment:
If we are not going to use this byte, why don better add it in the bridge part and remove from here?